crates/interpreter/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Minor
 
 - Implement `From<Error>` for `wasefire_error::Error`
+- Simplify `Thread::new`
 
 ### Patch
 
```

Comment:
nit: This is only patch-level because it's an internal detail.

```suggestion

### Patch

- Only take the initial frame in `Thread::new()` 
```